Doc/library/zipfile.rst

```diff
@@ -377,6 +377,10 @@ ZipFile Objects
    the new entry. Similarly, *compresslevel* will override the constructor if
    given.
    The archive must be open with mode ``'w'``, ``'x'`` or ``'a'``.
+   The *strict_timestamps* keyword argument, when set to ``False``, allows to
+   zip files older than 1980-01-01 at the cost of setting the
+   timestamp to 1980-01-01.
+   Same behavior occures with files older than 2107-12-31.
```

Comment:
It is not the same behavior, it is not set to 1980-01-01 in that case, but rather to 2107-12-31.

Reply:
Maybe something like: "Similar behavior occurs with files older than 2107-12-31, the timestamp is also set to the limit."

Reply:
Sounds good.
I have also noticed I have 'older' where 'newer' should be.